api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models/PublishedDataSetEventApiModel.cs

```diff
@@ -74,5 +74,17 @@ public class PublishedDataSetEventsApiModel {
         [DataMember(Name = "triggerId", Order = 8,
             EmitDefaultValue = false)]
         public string TriggerId { get; set; }
+
+        /// <summary>
+        /// Pending alarm settings
+        /// </summary>
+        [DataMember(Name = "pendingAlarms", Order = 9)]
+        public PendingAlarmsOptionsApiModel PendingAlarms { get; set; }
+
+        /// <summary>
+        /// Simple event Type id
+        /// </summary>
+        [DataMember(Name = "typeDefinitionId", Order = 10)]
+        public string TypeDefinitionId { get; set; }
```

Comment:
>TypeDefinitionId [](http://example.com/codeflow?start=22&length=16)

What exactly is this 'TypeDefinitionId'? 
If it is an event type id like in the comment the name is misleading, because in SimpleAttributeOperandModel class there is a 'TypeDefinitionId' that is a nodeId.
If it is a nodeId in PublishedDataSetEventApiModel class there is already a nodeId property